addons/account/models/account.py

```diff
@@ -1134,6 +1143,8 @@ class AccountReconcileModel(models.Model):
         ('fixed', 'Fixed'),
         ('percentage', 'Percentage of amount')
         ], string="Second Amount type",required=True, default='percentage')
+    second_tax_price_include = fields.Boolean(string='Is Second Tax Price Include', related='second_tax_id.price_include')
```

Comment:
store=False, no need of related

Reply:
The related is needed to implement:

> if the tax is already price included==True, then the boolaen is readonly (but True) to show the user what it will do

... in the view.